### PR TITLE
CORE-307 - Include MAN Into UK Network Data

### PR DESCRIPTION
**Pull request by CalumTowers**

### CODE REVIEW

File: app/Models/NetworkData/Atc.php [hunk 1]
```diff
@@ -187,6 +187,7 @@ public static function scopeIsUK($query)
                 ->orWhere('callsign', 'LIKE', "STC\_%")
                 ->orWhere('callsign', 'LIKE', "LON\_%")
                 ->orWhere('callsign', 'LIKE', "LTC\_%")
+                ->orWhere('callsign', 'LIKE', 'MAN\_%')
                 ->orWhere('callsign', 'LIKE', 'EGGX%')
                 ->orWhere('callsign', 'LIKE', 'EGTT%')
                 ->orWhere('callsign', 'LIKE', 'EGPX%');
```
Comment (AxonC) [inline app/Models/NetworkData/Atc.php]:
```suggestion
                ->orWhere('callsign', 'LIKE', 'EGPX%');
                ->orWhere('callsign', 'LIKE', 'LXGB_%');
                ->orWhere('callsign', 'LIKE', 'LCRA_%');
                ->orWhere('callsign', 'LIKE', 'FHAW_%');
```

Comment (AxonC) [reply app/Models/NetworkData/Atc.php]: I noticed when I did the manual queries that we were missing some of the minor positions. Might as well add them whilst we are at it!

File: app/Models/NetworkData/Atc.php [hunk 2]
```diff
@@ -187,9 +187,16 @@ public static function scopeIsUK($query)
                 ->orWhere('callsign', 'LIKE', "STC\_%")
                 ->orWhere('callsign', 'LIKE', "LON\_%")
                 ->orWhere('callsign', 'LIKE', "LTC\_%")
+                ->orWhere('callsign', 'LIKE', 'MAN\_%')
                 ->orWhere('callsign', 'LIKE', 'EGGX%')
                 ->orWhere('callsign', 'LIKE', 'EGTT%')
-                ->orWhere('callsign', 'LIKE', 'EGPX%');
+                ->orWhere('callsign', 'LIKE', 'EGPX%')
```
Comment (atoff) [inline app/Models/NetworkData/Atc.php]: This shouldn't be necessary due to the `EG%` rule above

**Pull request merged**